src/components/data-table-v2/_data-table-v2-core.scss

```diff
@@ -170,3 +170,296 @@
     background-color: rgba($brand-02, 0.1);
   }
 }
+
+@mixin data-table-v2-core--x {
+  //================================================
+  //MAIN CONTAINER
+  //================================================
+  .#{$prefix}--data-table-v2-container {
+    min-width: rem(600px);
+    padding-top: $spacing-3xs; // allow space for focus styles
+
+    .#{$prefix}--search-input {
+      background-color: transparent;
+    }
+  }
+
+  //================================================
+  //TABLE HEADER
+  //================================================
+  .#{$prefix}--data-table-v2-header {
+    background: $ui-01;
+    height: rem(88px);
+    color: $text-01;
+    @include font-size('20');
```

Comment:
Style nit: mixins on top:

```scss
  .#{$prefix}--data-table-v2-header {
    @include font-size('20');

    background: $ui-01;
    height: rem(88px);
    color: $text-01;
}
```

As an aside, any type prescription should be using our `type-style` mixin 👍

Reply:
Just fixed this. When I started this they still didn't have type tokens worked out, but it's updated now.